screens/scraping_test.go

```diff
@@ -317,7 +317,7 @@ func TestBuildWorkerPool(t *testing.T) {
 			hasScrapedImageFunc: func(rom string) bool {
 				return false
 			},
-			expectedEvents: []string{"Scrapped game1", "Scraping finished.", "Success: 1", "Failed: 0", "Skipped: 0"},
+			expectedEvents: []string{"scraped game1", "Scraping finished.", "Success: 1", "Failed: 0", "Skipped: 0"},
```

Comment:
It seems this test is still failing

```suggestion
			expectedEvents: []string{"Scraped game1", "Scraping finished.", "Success: 1", "Failed: 0", "Skipped: 0"},
```